### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by SardanDev** [Chatterino/chatterino2#4453] [Ideas]
Good evening, 
I propose you some ideas of adding which could be well:
- A possibility to be redirected to the chat of the person who was raided. (In particular to help in the raid)
- To be able to put a note on a person like the modos.
- To save the messages of the people in a log file which to put with that the messages on the card of the user. (Maybe it already exists but I didn't find the way to configure it)
- Have a space for channel points, not through the API but just a browser window because it's a shame not to be able to vote on predictions or to be able to launch them as modos (so can be the same for the player).
- Be able to put emoji in favorites.

:)

**Comment by Felanbird** [comment]
> Have a space for channel points, not through the API but just a browser window because it's a shame not to be able to vote on predictions or to be able to launch them as modos (so can be the same for the player).

We're not embedding anything from browser chat into Chatterino,

**Reply by SardanDev** [reply]
Okay

**Comment by Felanbird** [comment]
> A possibility to be redirected to the chat of the person who was raided. (In particular to help in the raid)

This was suggested in the discord, but I just want to note that this wouldn't _help_ the raid in any way, if you are not apart of the browser raid you don't count as a viewer who raided.

**Reply by SardanDev** [reply]
I know but just say hello

**Reply by SardanDev** [reply]
And know who is the channel who received the raid because some times it said but not always

**Reply by iProdigy** [reply]
Due to changes made by twitch (`HOSTTARGET` is no longer sent), we don't have a official scalable solution to reading outbound raids.

The only official approach we can take is eventsub websockets (subscribed to `channel.raid`) - but we'd be limited to a total of 10 channels.

There *is* an unofficial way to read raid events via pubsub, but this is unlikely to be implemented
